### PR TITLE
Update people.json

### PR DESCRIPTION
**Pull request by TheresaTeng**
Updated Theresa's interest.

### CODE REVIEW

File: people.json [hunk 1]
```diff
@@ -141,24 +141,28 @@
...
             "Application design",
-            "ASP.NET",
+            "ASP.NET Blazor",
+            "ASP.NET MVC",
             "Business analysis",
             "C# (programming language)",
+            "Django",
             "HTML",
             "JavaScript",
+            "Jira",
```
Comment (aharwood2) [inline people.json:152]: "Jira" should be "Jira (software)"

File: people.json [hunk 2]
```diff
@@ -141,24 +141,28 @@
...
             "JQuery",
             "Microsoft Windows",
-            "Model-view controller",
-            "Omeka",
+            "Microsoft SQL",
             "Project management",
+            "PostgreSQL",
             "SQL",
             "Visual Studio",
+            "Wagtail",
```
Comment (aharwood2) [inline people.json:160]: Should be "Wagtail (CMS)"

**Review by aharwood2 — CHANGES_REQUESTED** [review]
Sorry for deleting this branch -- I didn't see the open PR! Some changes required based on the controlled vocab really. We need an auto check for this really.

File: people.json [hunk 3]
```diff
@@ -141,24 +141,28 @@
 
     "Theresa Teng": {
         "interests": [
-            "AngularJS",
             "Application design",
-            "ASP.NET",
+            "ASP.NET Blazor",
```
[the remark this answers is not in the document]
Comment (TheresaTeng) [reply people.json]: Thank you. All the terms you mentioned have been changed.

**Review by aharwood2 — APPROVED** [review]
Changes to vocab all made. Happy for you to merge.

**Pull request merged**